docs/CONTRIBUTING.md

```diff
@@ -31,10 +31,18 @@ from onnx root dir should work.
 
 ## Generated operator documentation
 
-[Operator docs in Operators.md](Operators.md) are automatically generated based on C++ operator definitions and backend Python snippets. To refresh these docs, run the following commands from the repo root and commit the results. Note `ONNX_ML=0` updates Operators.md whereas `ONNX_ML=1` updates Operators-ml.md:
+[Operator docs in Operators.md](Operators.md) are automatically generated based on C++ operator definitions and backend Python snippets. To refresh these docs, run the following commands from the repo root and commit the results. 
 
+First, set `ONNX_ML` environment variable. `ONNX_ML=0` updates Operators.md whereas `ONNX_ML=1` updates Operators-ml.md:
+
+For Windows:
 ```
+# Windows
 set ONNX_ML=0
+
+# UNIX
+# export ONNX_ML=0
```

Comment:
Is that # in the second line a typo or intentional?

Reply:
It was a result of my failed attempt at fixing non-signed-off commit issue. My git got messed up, I'm going to open a new, clean PR. Sorry for inconvenience.